### DISCUSSION TITLE
Navigation

### DISCUSSION THREAD

**Discussion by spykee2005** [BimmerGestalt/AAIdrive#563] [Q&A]
Hi All

In the App description, I found this:

"Car Navigation Integration

Android Navigation Intents and buttons can be handled by the connected car's navigation

Google Maps can share destinations to the connected car's navigation"

Can you explain how does it work? 

What does it means: Android navigation Intents? 

Thanks a lot

**Comment by hufman** [comment]
Good morning, and thanks for the question! Android Intents are the general method through which Android apps can request another app do something. Various Android apps can show a button saying "Navigate to [this place]", like a car charger map or a food ordering app. Instead of each app needing to include a full map and navigation experience, they just request that the system opens whatever navigation app the user has installed. They can also hardcode the destination, for example Google Maps always opens the Google Navigation app, but by default these Intents open an app chooser or pick the default map app that was last opened.

When AAIdrive is connected to a car with Navigation onboard, the car will show up as an option in this App Chooser to handle Navigation Intents, same as if you had Google Maps and Waze installed at the same time.
